conda/recipes/libcuvs/recipe.yaml

```
@@ -0,0 +1,408 @@
+      - cuda-version =${{ cuda_version }}
+      - if: cuda_major == "11"
+        then:
+          - cuda-profiler-api =${{ cuda11_cuda_profiler_api_host_version }}
+          - libcublas =${{ cuda11_libcublas_host_version }}
+          - libcublas-dev =${{ cuda11_libcublas_host_version }}
+          - libcurand =${{ cuda11_libcurand_host_version }}
+          - libcurand-dev =${{ cuda11_libcurand_host_version }}
+          - libcusolver =${{ cuda11_libcusolver_host_version }}
+          - libcusolver-dev =${{ cuda11_libcusolver_host_version }}
+          - libcusparse =${{ cuda11_libcusparse_host_version }}
+          - libcusparse-dev =${{ cuda11_libcusparse_host_version }}
+        else:
+          - cuda-cudart-dev
+          - cuda-profiler-api
+          - libcublas-dev
+          - libcurand-dev
+          - libcusolver-dev
+          - libcusparse-dev
+      # These are used for bench-ann
+      - openblas
+      - if: linux64
+        then:
+          - libaio
+          - libboost-devel =1.87
+          - mkl-devel =2023
+
+outputs:
+  - package:
+      name: libcuvs
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+      script:
+        content: |
+          cmake --install cpp/build --component cuvs
+          cmake --install cpp/build --component c_api
+          cmake --install cpp/build --component hnswlib
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - librmm =${{ minor_version }}
+        - libraft-headers =${{ minor_version }}
+        - nccl ${{ nccl_version }}
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then:
+            - cuda-profiler-api =${{ cuda11_cuda_profiler_api_host_version }}
+            - libcublas =${{ cuda11_libcublas_host_version }}
+            - libcublas-dev =${{ cuda11_libcublas_host_version }}
+            - libcurand =${{ cuda11_libcurand_host_version }}
+            - libcurand-dev =${{ cuda11_libcurand_host_version }}
+            - libcusolver =${{ cuda11_libcusolver_host_version }}
+            - libcusolver-dev =${{ cuda11_libcusolver_host_version }}
+            - libcusparse =${{ cuda11_libcusparse_host_version }}
+            - libcusparse-dev =${{ cuda11_libcusparse_host_version }}
+          else:
+            - cuda-cudart-dev
+            - cuda-profiler-api
+            - libcublas-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+      run:
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - libraft-headers =${{ minor_version }}
+        - nccl
+        - if: cuda_major == "11"
+          then:
+            - cuda-cudart
+            - libcublas
+            - libcurand
+            - libcusolver
+            - libcusparse
```

I am guessing we need `cudatoolkit` on CUDA 11 and this list of math libraries on CUDA 12+?
```suggestion
        - if: cuda_major == "11"
          then:
            - cudatoolkit
          else:
            - cuda-cudart
            - libcublas
            - libcurand
            - libcusolver
            - libcusparse
```

---

You are right.  I am not sure I like that this passed tests like this